src/main/api/2/studio-api-2.yaml

```diff
@@ -7293,6 +7295,65 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /loggers:
+    get:
+      tags:
+        - loggers
+      summary: Get all loggers and its configured priority levels
+      description: Required permission "view_log_levels or configure_log_levels"
+      operationId: loggersGetAll
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  results:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/LoggerConfiguredLevel'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /loggers/log_level:
+    post:
+      tags:
+        - loggers
+      summary: Update logger priority level
+      description: Required permission "configure_log_levels"
+      operationId: loggersUpdateLevel
+      requestBody:
+        description: parameters to update the logger level
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LoggerConfiguredLevel'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  result:
+                    $ref: '#/components/schemas/LoggerConfiguredLevel'
```

Comment:
LoggerConfig

(it's general enough to add more params if we want, and specific enough to work for this particular case)